README.md

```diff
@@ -10,19 +10,14 @@ ANTARES cdci plugin is distributed under the terms of The MIT License.
 
 Who's responsible?
 -------------------
-Andrea Tramacere
+Denys Savchenko, Andrea Tramacere
 
 ISDC Data Centre for Astrophysics, Astronomy Department of the University of Geneva, Chemin d'Ecogia 16, CH-1290 Versoix, Switzerland
 
```

Comment:
@andriineronov I am about this line

Reply:
Remove "ISDC Data Centre for Astrophysics," it does not exist

Reply:
we could really also do these changes in a separate PR, it's not very clear if we are ok with the changes suggested here.